configs/inspection/Idealista.xml

```diff
@@ -1,10 +1,9 @@
-<profile version="1.0">
+<inspections profile_name="Idealista" version="1.0">
   <option name="myName" value="Idealista" />
   <inspection_tool class="AbstractClassNamingConvention" enabled="true" level="ERROR" enabled_by_default="true" />
   <inspection_tool class="AndroidLintContentDescription" enabled="false" level="WARNING" enabled_by_default="false" />
-  <inspection_tool class="AndroidLintWrongConstant" enabled="false" level="ERROR" enabled_by_default="false" />
   <inspection_tool class="AnonymousClassComplexity" enabled="true" level="ERROR" enabled_by_default="true">
-    <option name="m_limit" value="5" />
+    <option name="m_limit" value="3" />
```

Comment:
So for the CommandUi example we just need a value of "3" isn't it?

I think this value fits good with our background-ui-error method scheme, I don't know if it could crash with other anonymous classes or if we use anonymous classes with more complexity right now in our projects...

Anyway I think a value of 5 means a anonymous class with too many responsability for be a concrete class (regarding Android framework limitations itself, maybe it has anonymous classes with a complexity bigger than 3...I can't remember right now)